frontend/model/contracts/group.js

```diff
@@ -459,7 +460,7 @@ sbp('chelonia/defineContract', {
       validate: objectMaybeOf({
         settings: objectMaybeOf({
           // TODO: add 'groupPubkey'
-          groupName: string,
+          groupName: stringMax(GROUP_NAME_MAX_CHAR, 'groupName'),
```

Comment:
I thought passing `key` here and displaying in the error message would help with faster debugging.

Reply:
Seems like a good idea 👍